api/tests/unit/domain/usecases/find-association-between-user-and-organization-student_test.js

```diff
@@ -0,0 +1,101 @@
+const { expect, sinon, domainBuilder, catchErr } = require('../../../test-helper');
+const usecases = require('../../../../lib/domain/usecases');
+const Student = require('../../../../lib/domain/models/Student');
+const { CampaignCodeError, UserNotAuthorizedToAccessEntity } = require('../../../../lib/domain/errors');
+const campaignRepository = require('../../../../lib/infrastructure/repositories/campaign-repository');
+const studentRepository = require('../../../../lib/infrastructure/repositories/student-repository');
+
+describe('Unit | UseCase | find-association-between-user-and-organization-student', () => {
+
+  let studentReceivedStub;
+  let getCampaignStub;
+  let student;
+  let organizationId;
+  let userId;
+  let campaignCode;
+
+  beforeEach(() => {
+    userId = domainBuilder.buildUser().id;
+    organizationId = domainBuilder.buildOrganization().id;
+    campaignCode = domainBuilder.buildCampaign({ organizationId }).code;
+    student = domainBuilder.buildStudent({ organizationId, userId });
+    getCampaignStub = sinon.stub(campaignRepository, 'getByCode').throws('unexpected call');
+    studentReceivedStub = sinon.stub(studentRepository, 'findOneByUserIdAndOrganizationId').throws('unexpected call');
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('There is a student linked to the given userId', () => {
+
+    it('should call findOneByUserIdAndOrganizationId', async () => {
+      // given
+      getCampaignStub.withArgs(campaignCode).resolves({ organizationId });
+      studentReceivedStub.resolves({});
+
+      // when
+      await usecases.findAssociationBetweenUserAndOrganizationStudent({ authenticatedUserId: userId, requestedUserId: userId, campaignCode });
+
+      // then
+      expect(studentReceivedStub).have.been.calledOnce;
```

Comment:
il manque le `to`

Reply:
Tous les petits mots au milieu ne servent qu'à la lecture pour les développeurs. Tu peux tous les enchaîner sans que cela ne veuille rien dire, cela n'aura aucun impact.

Reply:
mais j'ai rajouté le "to" pour la lecture ;)

Reply:
ok merci pour l'info !